package.json

```diff
@@ -36,6 +36,7 @@
     "@positron/stack-trace": "1.0.0"
   },
   "devDependencies": {
+    "cheerio": "^1.0.0-rc.2",
```

Comment:
RC versions of library as dev dependency is okay right?

Reply:
Should be fine, all depends on how the author of that library handles their versioning. As you mentioned though this is a dev dependency only needed for testing, so very minimal risk if any.